diff -p -up shared-mime-info-0.80/freedesktop.org.xml.in.vhdl shared-mime-info-0.80/freedesktop.org.xml.in
--- shared-mime-info-0.80/freedesktop.org.xml.in.vhdl	2010-10-02 13:18:49.000000000 +0200
+++ shared-mime-info-0.80/freedesktop.org.xml.in	2010-10-02 13:18:49.000000000 +0200
@@ -3063,6 +3063,7 @@ command to generate the output files.
   <mime-type type="application/x-x509-ca-cert">
     <_comment>DER/PEM/Netscape-encoded X.509 certificate</_comment>
     <generic-icon name="text-x-generic"/>
+    <sub-class-of type="text/plain"/> 
     <glob pattern="*.der"/>
     <glob pattern="*.crt"/>
     <glob pattern="*.cert"/>
