diff -up shared-mime-info-2.3/src/update-mime-database.cpp.omv~ shared-mime-info-2.3/src/update-mime-database.cpp
--- shared-mime-info-2.3/src/update-mime-database.cpp.omv~	2023-10-14 22:34:44.566403581 +0200
+++ shared-mime-info-2.3/src/update-mime-database.cpp	2023-10-14 22:34:58.605530044 +0200
@@ -2158,7 +2158,7 @@ static void check_in_path_xdg_data(const
 
 	env = getenv("XDG_DATA_DIRS");
 	if (!env)
-		env = "/usr/local/share/"PATH_SEPARATOR"/usr/share/";
+		env = "/usr/local/share/" PATH_SEPARATOR "/usr/share/";
 	dirs = g_strsplit(env, PATH_SEPARATOR, 0);
 	g_return_if_fail(dirs != NULL);
 	for (n = 0; dirs[n]; n++)
