diff -urN shared-mime-info-1.3/freedesktop.org.xml shared-mime-info-1.3-patched/freedesktop.org.xml
--- shared-mime-info-1.3/freedesktop.org.xml	2014-04-08 23:51:03.000000000 +1100
+++ shared-mime-info-1.3-patched/freedesktop.org.xml	2014-10-01 23:28:42.438237428 +1100
@@ -7197,8 +7197,8 @@
     <comment xml:lang="zh_TW">原生 CD 映像</comment>
     <sub-class-of type="application/x-raw-disk-image"/>
     <alias type="application/x-iso9660-image"/>
-    <glob pattern="*.iso"/>
-    <glob pattern="*.iso9660"/>
+    <glob weight="60" pattern="*.iso"/>
+    <glob weight="60" pattern="*.iso9660"/>
   </mime-type>
   <mime-type type="application/x-cdrdao-toc">
     <comment>CD Table Of Contents</comment>
diff -urN shared-mime-info-1.3/freedesktop.org.xml.in shared-mime-info-1.3-patched/freedesktop.org.xml.in
--- shared-mime-info-1.3/freedesktop.org.xml.in	2014-04-08 23:46:11.000000000 +1100
+++ shared-mime-info-1.3-patched/freedesktop.org.xml.in	2014-10-01 23:28:33.924237041 +1100
@@ -1499,8 +1499,8 @@
     <_comment>raw CD image</_comment>
     <sub-class-of type="application/x-raw-disk-image"/>
     <alias type="application/x-iso9660-image"/>
-    <glob pattern="*.iso"/>
-    <glob pattern="*.iso9660"/>
+    <glob weight="60" pattern="*.iso"/>
+    <glob weight="60" pattern="*.iso9660"/>
   </mime-type>
   <mime-type type="application/x-cdrdao-toc">
     <_comment>CD Table Of Contents</_comment>
